gwpy/signal/qtransform.py

```diff
@@ -288,7 +289,7 @@ def farray(self):
     def whitening_duration(self):
         """The recommended data duration required for whitening
         """
-        return 2 ** (round(log(self.q / (2 * self.frange[0]), 2)))
+        return round_to_power(self.q / (2 * self.frange[0]))
```

Comment:
I think passing the keywords might make the usage a little clearer in this case.

```suggestion
        return round_to_power(self.q / (2 * self.frange[0]), base=2, which=None)
```

Reply:
@duncanmmacleod, this is done in the latest commit